### DISCUSSION TITLE
API - karty podarunkowe

### DISCUSSION THREAD

**Discussion by kamilbo** [allegro/allegro-api#8250] [Suggestions - rejected]
Nie udało mi się znaleźć w dokumentacji żadnych endpointów związanych z kartami podarunkowymi.

Bardzo przydatne byłyby dla mnie dwie funkcjonalności :
1. Możliwość zamówienia / zakupienia kart podarunkowych o różnych nominałach w różnych ilościach.
2. Możliwość sprawdzenia, czy karty zakupione wcześniej zostały już zużyte, czy też nadal można je wykorzystać.

Czy jest jakaś możliwość rozwiązania tych problemów przy użyciu istniejącego API ?

**Comment by Lukasz-Zurek** [comment]
Nie mamy tego typu funkcjonalności w API. Niemniej jednak dziękuję Tobie za przesłane propozycje. Twój wątek przenoszę do [Discussions](https://github.com/allegro/allegro-api/discussions), czyli w miejsce dedykowane do przekazywania sugestii dotyczących rozwoju Allegro API.

**Comment by Lukasz-Zurek** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona" - nie planujemy sugerowanej zmiany. Propozycja dotyczy niejako obszaru zakupów, a takich funkcjonalności w API również narazie nie planujemy.
